Sources/PrimerSDK/Classes/Core/PrimerHeadlessUniversalCheckout/Composable/NativeUI/PayPal/PayPalValidationComponent.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  PayPalValidationComponent.swift
+//  PrimerSDK
+//
+//  Created by Niall Quinn on 12/02/24.
+//
+
+import Foundation
+
+struct PayPalValidationComponent: NativeUIValidateable {
+    let paymentMethodType = PrimerPaymentMethodType.payPal.rawValue
+
+    func validatePaymentMethod() throws {
+        if PrimerSettings.current.paymentMethodOptions.urlScheme == nil {
+            let err = PrimerError.invalidUrlScheme(urlScheme: nil,
```

Comment:
I think we should decide whether we store errors as `err` or `error`, I vote for `error`. WDYT?